example-embedded-app/src/components/Link.tsx

```diff
@@ -10,7 +10,10 @@ const Anchor = styled("a")({});
 
 interface NextLinkComposedProps
   extends Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, "href">,
-    Omit<NextLinkProps, "href" | "as" | "onClick" | "onMouseEnter"> {
+    Omit<
+      NextLinkProps,
+      "href" | "as" | "onClick" | "onMouseEnter" | "onTouchStart"
```

Comment:
This was causing a typescript error because both extends types defined `onTouchStart` with different shapes.